src/bootstrap/Cargo.toml

```diff
@@ -37,14 +37,9 @@ test = false
 # Most of the time updating these dependencies requires modifications to the
 # bootstrap codebase(e.g., https://github.com/rust-lang/rust/issues/124565);
 # otherwise, some targets will fail. That's why these dependencies are explicitly pinned.
-<<<<<<< HEAD
 
+# Ferrocene note: upstream downgraded to "1.1.22", but "1.2.0" version works for us
```

Comment:
What is the rationale for diverging from upstream here? There has been a lot of discussions and problems around the `cc` crate, and we should just follow what upstream does there.

Reply:
there been too much back n forth (upstream bumps, we pin to avoid ci fail, we fix that fail and bump to match upstream, upstream breaks us, we fix, and now upstream downgrades), so this was me seeking calm

anyways, this removes the divergence 5367c72c4479c9eac777625a72f267b5381b94c4